### PR TITLE
magento/merchdocs#60: Update product review information

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request updates product review information:
 - Enable Magento Product Reviews
 - Moderate Reviews
 - Ratings

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [x] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/marketing/magento-product-reviews-config.html
https://docs.magento.com/user-guide/marketing/magento-product-reviews-moderate.html
https://docs.magento.com/user-guide/stores/attributes-rating.html

### CODE REVIEW

File: src/marketing/magento-product-reviews-moderate.md [hunk 1]
```diff
@@ -4,13 +4,33 @@ title: Moderate Reviews
...
 1. In the list, click a pending review to view the details and edit if necessary.
 
 1. Change the **Status** according to your assessment:
 
    - To approve a pending review, select `Approved`.
-   - To reject a review, select `Not Approved`.
+
+   - To reject a review, select `Not Approved`. Unapproved reviews disappear from the list of **Pending Reviews** page.
+
+1. Also, the admin can moderate the **Detailed Rating**, **Nickname**, **Summary of Review** fields.
```
Comment (leslietilling) [inline src/marketing/magento-product-reviews-moderate.md:17]:
```suggestion
1. If needed, change the values for **Detailed Rating**, **Nickname**, and **Summary of Review** .
```

File: src/stores/attributes-rating.md [hunk 2]
```diff
@@ -39,3 +39,8 @@ _Ratings_
     _Rating Visibility_
 
 1. When complete, click <span class="btn">Save Rating</span>.
+
+1. The average rating for all reviews is displayed for each product on the catalog product grid page.
```
Comment (leslietilling) [inline src/stores/attributes-rating.md:43]: This is not really a step, more of a final result
```suggestion
   The average rating for all reviews is displayed for each product on the catalog product grid page.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Minor edits

**Comment by ZhovnirMaria** [comment]
Thanks, @leslietilling 😊
Done✅

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**